packages/ember-svg-jar/addon/helpers/svg-jar.js

```diff
@@ -1,6 +1,8 @@
 import { helper } from '@ember/component/helper';
 import makeSVG from 'ember-svg-jar/utils/make-svg';
-import icons from '../inlined';
```

Comment:
I think you needed `import * as icons from ...`

Reply:
I switched back to importSync but you're probably right